### ISSUE TITLE
bring back aim assist

### ISSUE THREAD

**Issue by Sterben235** [MeteorDevelopment/meteor-client#3659]
### Describe the feature

aim assist was removed in dev build #1830 it's a great feature for a subtle pvp advantage and is surly missed

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by Pxddyk45** [comment]
get better at aiming

**Comment by Shrecknt** [comment]
get better at aiming

**Comment by Big-Iron-Cheems** [comment]
I'll be honest chief: 
the only pleople who use aim assist are cheaters that try to hide from anti cheats on servers that don't allow Meteor.
I'm quite sure this was obvious to anyone with more than one neuron.

If you're using a blatant tool like Meteor, then aim assist is useless, considering kill aura exists.
And no, we're not going to add it back most certainly.

TL;DR: you're probably getting clapped by kids on hypixel half your age, and feel the need to use cheats to beat them.

**Comment by BDARmech** [comment]
I'd say low blow, but their armor didn't matter. "D

**Comment by 6tanooki9** [comment]
but still why not have it it is nice to have on rpg servers along with autowalk
